lib/primer/classify.rb

```diff
@@ -167,12 +165,6 @@ def extract_value(memo, key, val, breakpoint)
           BOOLEAN_MAPPINGS[key][:mappings].each do |m|
             memo[:classes] << m[:css_class] if m[:value] == val && m[:css_class].present?
           end
-        elsif key == BG_KEY
-          if val.to_s.start_with?("#")
-            memo[:styles] << "background-color: #{val};"
-          else
```

Comment:
I was going to still support this at the top, but I did a search on dotcom and this is used less than 20 times. I think I'd rather drop the support and convert `bg: "#000"` to `style: "background-color: #000;"`

Reply:
https://github.com/github/github/pull/193583 will take care of this issue